server/src/main/java/com/defold/extender/TemplateExecutor.java

```diff
@@ -16,7 +16,12 @@ public class TemplateExecutor {
 
     public String execute(String template, Map<String, Object> context) {
         try {
-        	return Mustache.compiler().compile(template).execute(context);
+        	String result = Mustache.compiler().compile(template).execute(context);
+            while (!result.equals(template)) {
+                template = result;
+                result = Mustache.compiler().compile(template).execute(context);
+            }
```

Comment:
A problem we previously had was that if one variable contained another expression, we couldn't resolve that:

```
env:
  a: "foo"
  b: "{{env.a}}"
  c: "{{env.b}}"
```